src/NLog.Targets.Syslog/Policies/ReplaceComputedValuePolicy.cs

```diff
@@ -25,11 +25,12 @@ public bool IsApplicable()
 
         public string Apply(string s, string searchFor)
         {
-            if (string.IsNullOrEmpty(searchFor) || string.IsNullOrEmpty(replaceWith) || s.Length == 0)
+            if (s.Length == 0 || string.IsNullOrEmpty(searchFor))
```

Comment:
Shouldn't a check for `replaceWith == null` be present?

Reply:
[Yes](https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.replace?view=netframework-4.5.2#System_Text_RegularExpressions_Regex_Replace_System_String_System_String_System_String_)

Reply:
Yes: otherwise an exception will the [thrown](https://docs.microsoft.com/en-us/dotnet/api/system.text.regularexpressions.regex.replace?view=netframework-4.5.2#System_Text_RegularExpressions_Regex_Replace_System_String_System_String_System_String_) by `Regex.Replace`